### PR TITLE
🎨 Add 2 rclone e2e tests

### PR DESCRIPTION
**Pull request by mguidon**
## What do these changes do?

Adds two feature tests for `rclone` functionality.

### CODE REVIEW

File: tests/e2e/tutorials/rclone_large.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+// node rclone_small.js [url] [--user user] [--pass password] [--demo]
```
Comment (odeimaiz) [inline tests/e2e/tutorials/rclone_large.js:1]:
```suggestion
// node rclone_large.js [url] [--user user] [--pass password] [--demo]
```

File: tests/e2e/tutorials/rclone_large.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    const studyData = await tutorial.openTemplate(1000);
+    studyId = studyData["data"]["uuid"];
+
+    const workbenchData = utils.extractWorkbenchData(studyData["data"]);
+    await tutorial.waitForServices(
+      workbenchData["studyId"],
+      [workbenchData["nodeIds"][1], workbenchData["nodeIds"][2]],
+      startTimeout,
+      false
+    );
```
Comment (odeimaiz) [inline tests/e2e/tutorials/rclone_large.js:34]: Here you are waiting for the 2nd and 3rd services... I guess you want to wait for all of them.

File: tests/e2e/tutorials/rclone_small.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    const studyData = await tutorial.openTemplate(1000);
+    studyId = studyData["data"]["uuid"];
+
+    const workbenchData = utils.extractWorkbenchData(studyData["data"]);
+    await tutorial.waitForServices(
+      workbenchData["studyId"],
+      [workbenchData["nodeIds"][1], workbenchData["nodeIds"][2]],
+      startTimeout,
+      false
+    );
```
Comment (odeimaiz) [inline tests/e2e/tutorials/rclone_small.js:34]: Same here

File: tests/e2e/tutorials/rclone_large.js (hunk 2 above)
Comment (mguidon) [reply tests/e2e/tutorials/rclone_large.js:34]: indeed

**Review by pcrespov — APPROVED** [review]

**Pull request merged**